platform/lib/sdk/src/main/kotlin/io/hamal/lib/sdk/api/Code.kt

```diff
@@ -3,11 +3,36 @@ package io.hamal.lib.sdk.api
 import io.hamal.lib.domain.vo.CodeId
 import io.hamal.lib.domain.vo.CodeValue
 import io.hamal.lib.domain.vo.CodeVersion
+import io.hamal.lib.http.HttpTemplate
+import io.hamal.lib.sdk.fold
 import kotlinx.serialization.Serializable
 
 @Serializable
 data class ApiCode(
     val id: CodeId,
     val value: CodeValue,
     val version: CodeVersion
-)
\ No newline at end of file
+)
+
+interface ApiCodeService {
+    fun get(codeId: CodeId): ApiCode
+    fun get(codeId: CodeId, codeVersion: CodeVersion): ApiCode
+}
```

Comment:
```suggestion
interface ApiCodeService {
    fun get(codeId: CodeId, codeVersion: CodeVersion? = null): ApiCode
}
```

Try to make codeVersion to be an optional parameter